deps/build.jl

```diff
@@ -10,13 +10,14 @@ module DefaultDeps
     if !isdefined(@__MODULE__, :MINICONDA_VERSION)
         const MINICONDA_VERSION = "3"
     end
-    if !isdefined(@__MODULE__, :ROOTENV)
-        const ROOTENV = joinpath(Main.condadir, MINICONDA_VERSION)
+    if !isdefined(@__MODULE__, :CONDA_HOME)
+        const CONDA_HOME = Main.condadir
     end
 end
 
-ROOTENV = get(ENV, "CONDA_JL_HOME", DefaultDeps.ROOTENV)
+CONDA_HOME = get(ENV, "CONDA_JL_HOME", DefaultDeps.CONDA_HOME)
```

Comment:
If a user has set `CONDA_JL_HOME` to say `/Users/omus/.julia/conda/3` then a new root will be created as `/Users/omus/.julia/conda/3/$MINICONDA_VERSION`

Reply:
`CONDA_JL_HOME` is an option to use an existing conda environment.

Reply:
Ok, I believe I can revise this to work correctly with that functionality